code/modules/client/preference_setup/loadout/loadout_general.dm

```diff
@@ -155,4 +166,63 @@
 	toothpaste["toothpaste and blue toothbrush"] = /obj/item/storage/box/toothpaste
 	toothpaste["toothpaste and green toothbrush"] = /obj/item/storage/box/toothpaste/green
 	toothpaste["toothpaste and red toothbrush"] = /obj/item/storage/box/toothpaste/red
-	gear_tweaks += new/datum/gear_tweak/path(toothpaste)
\ No newline at end of file
+	gear_tweaks += new/datum/gear_tweak/path(toothpaste)
+
+/datum/gear/figure
+	display_name = "action figure (selection)"
+	description = "A \"Space Life\" brand action figure."
+	path = /obj/item/toy/figure
+
+/datum/gear/figure/New()
+	..()
+	var/list/figures = list()
+	for(var/figure in typesof(/obj/item/toy/figure/) - /obj/item/toy/figure)
+		var/obj/item/toy/figure/figure_type = figure
+		figures[initial(figure_type.name)] = figure_type
+	gear_tweaks += new/datum/gear_tweak/path(sortAssoc(figures))
+
+/datum/gear/plushie
+	display_name = "plushie (selection)"
+	path = /obj/item/toy/plushie
+
+/datum/gear/plushie/New()
+	..()
+	var/list/plushies = list()
+	for(var/plushie in subtypesof(/obj/item/toy/plushie/) - /obj/item/toy/plushie/therapy)
+		var/obj/item/toy/plushie/plushie_type = plushie
+		plushies[initial(plushie_type.name)] = plushie_type
+	gear_tweaks += new/datum/gear_tweak/path(sortAssoc(plushies))
+
+/datum/gear/toy
+	display_name = "toy (selection)"
```

Comment:
I'd argue that they still don't make it moot. People can bring these things back home and I don't see why they wouldn't be able to bring a few little of these things with them to decorate their workspace with or keep in their locker. I haven't added the entire list of arcade prizes onto here.
That said, I am considering it might be odd with the stick horse. I'd still like to present the opportunity with other people though on what they can think of with regards to it. Just in case.